lib/guardian/group_guardian.rb

```diff
@@ -3,6 +3,15 @@
 #mixin for all guardian methods dealing with group permissions
 module GroupGuardian
 
+  # Creating Method
+  def can_create_group?(parent = nil)
```

Comment:
Looks like the `parent` parameter is never used.

Reply:
`guardian.ensure_can_create!(Group)` was being called from admin/groups_controller.rb, which would result in EnsureMagic calling `can_create_group` and passing in a param. Changing things to call `guardian.ensure_can_create_group!` allows us to get rid of the parent param.

👍